index.html

```diff
@@ -3,8 +3,43 @@
   <head>
     <meta charset="utf-8">
     <title>learne</title>
+    <link rel="stylesheet" href="app.css">
+    <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/jquery.slick/1.6.0/slick.css"/>
+    <script src="https://use.fontawesome.com/ac8a89b97e.js"></script>
+
   </head>
+
   <body>
 
+    <div id="headWrapper">
+      <header>
+        <ul>
+          <li><i class="fa fa-graduation-cap" aria-hidden="true"></i>
+            Learne</li>
+          <li>שיטת הלימוד</li>
+          <li>לקודחות מספרים</li>
+          <li> צור קשר </li>
+        </ul>
+      </header>
+    </div>
+
+    <!--slider-->
+    <div class="single-item" id="slider">
+      <div><img src="http://kenwheeler.github.io/slick/img/fonz1.png"></div>
+      <div><img src="http://kenwheeler.github.io/slick/img/fonz2.png"></div>
+      <div><img src="http://kenwheeler.github.io/slick/img/fonz3.png"></div>
+    </div>
+
+    <!--section3-->
+    <div id="sec3"></div>
```

Comment:
This is not a good name as ID.
Use a declarative name, as this div might have a different position one day in case we move things around or add another section before